expenses/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.0.3 on 2022-03-20 16:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Expenses',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('amount', models.IntegerField()),
+                ('year', models.IntegerField()),
+                ('category', models.CharField(choices=[('Rent', 'Rent'), ('Mortgage', 'Mortgage'),
+                                                       ('bills', 'Bills'), ('Transportation', 'Transportation'),
+                                                       ('Clothing', 'Clothing'), ('Healthcare', 'Healthcare'),
+                                                       ('Food', 'Food'), ('Insurance', 'Insurance'),
+                                                       ('Kids', 'Kids'), ('Culture', 'Culture'),
+                                                       ('Vacations', 'Vacations'), ('Other', 'Other')],
+                                              default='Other', max_length=32)),
+                ('month', models.CharField(choices=[('January', 'January'), ('February', 'February'),
```

Comment:
Another possible way is to use DateField instead of two different fields for month and year
Use whatever you think is better, I was just suggesting

Reply:
That is a good Idea